### DISCUSSION TITLE
Default R arguments

### DISCUSSION THREAD

**Discussion by shikokuchuo** [wlandau/crew#183] [General]
I saw your commits 24e8fd8 and 1a10017 adding the default 'r_arguments' `c("--no-save", "--no-restore")`.

They struck me as unnecessary given `?Rscript`:
> Options --no-echo --no-restore are always supplied: these imply --no-save.

Then I realised you use `processx` as a local launcher and you have your `bin` configured as "R.exe"/ "R". I think this can use "Rscript.exe" / "Rscript" as `mirai` itself does.

**Comment by wlandau** [comment]
I will think about it. Didn't realize R and Rscript had different default flags.

**Reply by shikokuchuo** [reply]
I've always just assumed that `Rscript` was the way to run things non-interactively. ChatGPT tells me: Rscript is a more lightweight and optimized way to execute R scripts in non-interactive sessions.

**Reply by wlandau** [reply]
Just switched to `Rscript`. Keeping `r_arguments = c("--no-save", "--no-restore")` in case we switch back.
